src/panels/lovelace/hui-root.ts

```diff
@@ -298,7 +291,7 @@ class HUIRoot extends hassLocalizeLitMixin(LitElement) {
       </app-header>
       <div id='view' class="${classMap({
         "tabs-hidden": this.lovelace!.config.views.length < 2,
-      })}" @rebuild-view='${this._debouncedConfigChanged}'></div>
+      })}" @ll-rebuild='${this._debouncedConfigChanged}'></div>
```

Comment:
Not sure how events work...
Will this get triggered every time the ll-rebuild event is fired, or is it caught and eaten by the handler in hui-view?

Reply:
If a card takes it, it will stop it from reaching the view. Only in edit mode we let it reach the view

Reply:
Good

Reply:
It's done by calling `event.stopPropagation()` in the event handler for the card.

https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation